lib/components/util/dropdown.tsx

```diff
@@ -0,0 +1,126 @@
+import { useIntl } from 'react-intl'
+import React, { useEffect, useRef, useState } from 'react'
+import styled from 'styled-components'
+
+type Props = {
+  children: JSX.Element
+  id: string
+  pullRight?: boolean
+  style?: { [key: string]: any }
+  title: JSX.Element
+}
+
+const DropdownButton = styled.a`
```

Comment:
This should effectively be a `button`. Bootstrap uses `a` generically but for dropdowns assigns a "button" role, which is redundant for our use.

Plus, buttons can be clicked using the keyboard using Enter or Space, whereas links can only be activated using "Enter" (unless otherwise programmed).

Reply:
I originally had a button, but it caused too many conflicts with the bootstrap nav bar styles. Will add a TODO.

Reply:
Did you add the TODO?

Reply:
Good catch thank you!